### ISSUE TITLE
An enhancement suggestion, and I'd like to contribute to this project

### ISSUE THREAD

**Issue by Caoxuyang** [hyperledger/caliper#92]
Hi there,
I'm an intern in Microsoft and I wonder can I try to extend the caliper to support Ethereum and Parity private chains? 
And this is the first step, if we can do with Eth and Parity, we can support most rpc platforms like EOS and so on. (My thought, maybe I'm wrong.)
And if I'm allowed, I's be happy to start the work and contribute to caliper!

Best Regards,
Xuyang

**Comment by Caoxuyang** [comment]
I've already started to work on it.

**Comment by aklenik** [comment]
@Caoxuyang Every contribution is welcome of course :) The [CONTRIBUTING.md](https://github.com/hyperledger/caliper/blob/master/CONTRIBUTING.md) file contains some guidelines, and also feel free to ask about implementation details in the [#caliper-contributors](https://chat.hyperledger.org/channel/caliper-contributors) RocketChat channel.

**Comment by Caoxuyang** [comment]
@aklenik Sorry for late reply, I was busy doing my intern project. :)
Now actually I've finished my extended version of Caliper which can give benchmark tests on Ethereum/Parity network. As long as I have a RPC address of the target network.
I implemented benchmark tests for both with or without smart contranct, and both in the same way which is signing the transaction locally and send signed transaction to the network.
My tps test result of Parity Network is around 3000. And I usually do tests like sending 200000 transactions totally. 
I do willing to contribute, and I will clean up my code first and see what't next.
Thanks a million!
